java/connector-node/risingwave-sink-iceberg/src/main/java/com/risingwave/connector/catalog/JniCatalogWrapper.java

```diff
@@ -100,6 +100,43 @@ public boolean tableExists(String tableIdentifier) {
         return catalog.tableExists(id);
     }
 
+    /**
+     * Checks if a namespace exists in the catalog.
+     *
+     * @param namespaceStr The namespace to check.
+     * @return true if the namespace exists, false otherwise.
+     */
+    public boolean namespaceExists(String namespaceStr) {
+        Namespace namespace;
+        if (namespaceStr == null) {
+            namespace = Namespace.empty();
```

Comment:
Should we return directly when the namespace is empty?

Reply:
I think we can let the underlying system to decide.